cluster1.py

```diff
@@ -112,12 +77,54 @@ def add_to_graph(v_table, costs, counter, found_vertices):
     return new_counter
 
 
-for i in range(n - K):
-    counter = add_to_graph(vertices, input_costs, counter, found_vertices)
+def read_input_costs(filename):
```

Comment:
This is a comment.  We are communicating.  Testing 123 testing 123 plzkthxbai

Reply:
nice job